raisinbread/instruments/eeg_acquisition_form.linst

```diff
@@ -0,0 +1,88 @@
+text{@}acquisition_software_other{@}If other, please specify
+select{@}recording_reference{@}Recording reference used{@}'left_mastoid'=>'left mastoid'{-}'right_mastoid'=>'right mastoid'{-}'left_earlobe'=>'left earlobe'{-}'right_earlobe'=>'right earlobe'{-}'physically_linked_mastoids'=>'physically linked mastoids'{-}'physically_linked_earlobes'=>'physically linked earlobes'{-}'cz'=>'Cz'{-}'tip_nose'=>'tip of the nose'{-}'balanced_non_cephalic_reference'=>'balanced non-cephalic reference'{-}'other'=>'Other'
+text{@}recording_reference_other{@}If other, please specify
+text{@}number_active_channels{@}Number of active channels
+select{@}number_active_channels_status{@}Number of active channels{@}NULL=>''{-}'not_answered'=>'Not Answered'
+select{@}eog_channels{@}EOG channels{@}'no_eog_channels'=>'no EOG channels'{-}'two_heog_channels_only'=>'two HEOG channels only'{-}'two_veog_channels_only'=>'two VEOG channels only'{-}'two_heog_two_veog_channels'=>'two HEOG and two VEOG channels'{-}'other'=>'Other'
+text{@}eog_channels_other{@}If other, please specify
+textarea{@}artefacted_channels{@}Artefacted channels
+header{@}{@}<b><h3>EEG Acquisition</h3></b>
+select{@}bad_electrodes_present{@}Were there any bad electrodes during impedances?{@}'yes'=>'Yes'{-}'no'=>'No'
+textarea{@}bad_electrodes{@}If Yes, please register all bad electrodes
+textarea{@}hardware_filters_applied{@}What hardware filters were applied before acquisition?
+textarea{@}software_filters_applied{@}What software filters were applied during acquisition?
+time{@}eeg_session_start_time{@}EEG session Start time
+select{@}eeg_session_start_time_status{@}EEG session Start time{@}NULL=>''{-}'dnk'=>'DNK'{-}'not_answered'=>'Not Answered'
+time{@}eeg_session_end_time{@}EEG session End time
+select{@}eeg_session_end_time_status{@}EEG session End time{@}NULL=>''{-}'dnk'=>'DNK'{-}'not_answered'=>'Not Answered'
+header{@}{@}<h3>Resting state task</h3>
+select{@}resting_state_eeg_acquired{@}Resting State EEG Acquired?{@}NULL=>''{-}'not_started'=>'Not Started'{-}'complete'=>'Complete'{-}'incomplete'=>'Incomplete'
+textarea{@}resting_state_eeg_incomplete_reason{@}If Incomplete, why were you unable to complete the task?
+select{@}resting_state_attempts{@}Number of attempts{@}NULL=>''{-}'1'=>'1'{-}'2'=>'2'{-}'3'=>'3'{-}'4'=>'4'{-}'more_than_5'=>'More than 5'
+select{@}resting_state_eeg_mood_rating{@}Resting State EEG Mood Rating{@}NULL=>''{-}'1'=>'1 - Screaming, fussed out'{-}'2'=>'2'{-}'3'=>'3'{-}'4'=>'4'{-}'5'=>'5 - Very calm the whole time'
+textarea{@}resting_state_eeg_mood_rating_comments{@}Comments on subject mood:
+textarea{@}resting_state_comments{@}Comments on the resting state:
+header{@}{@}<h3>Task 1</h3>
+select{@}task1_paradigm_used{@}Task 1 used?{@}'yes'=>'Yes'{-}'no'=>'No'
```

Comment:
What does "Task 1 used" mean ?  par rapport a "Task 1 eeg collected" -- 
Is it possible for the Task  to be used with no eeg acquired?   or for task eeg to be acquired without the Task to be "used" ?  What other word is more appropriate than "used"?